### PR TITLE
[MRG] update JOSS for sourmash 4.4

### PR DESCRIPTION
**Pull request by bluegenes**
draft for sourmash 4.x.

### CODE REVIEW

**Comment by bluegenes** [comment]
@ctb ready for review

File: paper.md [hunk 1]
```diff
@@ -74,21 +78,19 @@ bibliography: paper.bib
...
 
-MinHash sketches provide a lightweight way to store "signatures" of
-large DNA or RNA sequence collections, and then compare or search them
-using a Jaccard index [@Ondov:2015].  MinHash sketches can be used
-to identify samples, find similar samples, identify data sets with
-shared sequences, and build phylogenetic trees.
+FracMinHash sketching is lossy compression approach that represents
+data sets using a "fractional" sketch containing $1/S$ of the original 
+k-mers. Like other sequence sketching techniques (e.g. MinHash, [@Ondov:2015]), FracMinHash provides a lightweight way to store representations of large DNA or RNA sequence collections for comparison and search. Sketches can be used to identify samples, find similar samples, identify data sets with shared sequences, and build phylogenetic trees. FracMinHash sketches support estimation of overlap, bidirectional containment, and Jaccard similarity between data sets and is accurate even for data sets of different sizes.
```
Comment (ctb) [inline paper.md]:
```suggestion
k-mers. Like other sequence sketching techniques (e.g. MinHash, [@Ondov:2015]), FracMinHash provides a lightweight way to store representations of large DNA or RNA sequence collections for comparison and search. Sketches can be used to identify samples, find similar samples, identify data sets with shared sequences, and build phylogenetic trees. FracMinHash sketching supports estimation of overlap, bidirectional containment, and Jaccard similarity between data sets and is accurate even for data sets of very different sizes.
```

File: paper.md [hunk 2]
```diff
@@ -74,21 +78,19 @@ bibliography: paper.bib
...
-release, sourmash v4, is built on top of Rust and provides an
-experimental Rust interface.
+search, comparison, and analysis [@Pierce:2019]. sourmash's FracMinHash sketching supports fast and accurate sequence comparisons between datasets of different sizes [@gather], including petabase-scale database search [@branchwater]. From release 4.x, sourmash is built on top of Rust and provides an experimental Rust interface.
 
-MinHash sketches provide a lightweight way to store "signatures" of
-large DNA or RNA sequence collections, and then compare or search them
-using a Jaccard index [@Ondov:2015].  MinHash sketches can be used
-to identify samples, find similar samples, identify data sets with
-shared sequences, and build phylogenetic trees.
+FracMinHash sketching is lossy compression approach that represents
```
Comment (ctb) [inline paper.md]:
```suggestion
FracMinHash sketching is a lossy compression approach that represents
```

**Review by ctb — APPROVED** [review]
two suggestions => good to go!

**Comment by bluegenes** [comment]
merging this so I can work in the base JOSS update branch

**Pull request merged**